docs/_book.yaml

```diff
@@ -147,8 +147,6 @@ upper_tabs:
       - heading: "Google hardware"
       - title: "Getting started with QCS"
         path: /cirq/tutorials/google/start
-      - title: "Reservations"
```

Comment:
Is this because we can use the console rather than the colab? SGTM, but we should probably include new docs for this...that's on me :-)

Reply:
Yes, exactly because of that! Agreed - we probably should :)